com.unity.cinemachine/Runtime/Timeline/CinemachineTrack.cs

```diff
@@ -16,6 +16,12 @@ namespace Unity.Cinemachine
     [TrackColor(0.53f, 0.0f, 0.08f)]
     public class CinemachineTrack : TrackAsset
     {
+        [Tooltip("The priority controls the precedence that this track takes over other CinemachineTracks.  "
+            + "Tracks with higher priority will override tracks with lower priority.  If two " 
+            + "simultaneous tracks have the same priority, then the more-recently instanced track will "
+            + "take precedence.  Track priority is unrelated to Cinemachine Camera priority.")]
```

Comment:
I understand the need for visibility when possible. But should we change the code for that?

Reply:
I don't understand what you are asking